### DISCUSSION TITLE
v6.4.0

### DISCUSSION THREAD

**Discussion by Ghost-chu** [PBH-BTN/PeerBanHelper#549] [公告]
## 前言

大家好，各位期待已久的 BitComet 下载器支持终于到来了。  
在[彗星论坛](https://www.cometbbs.com/)、BitComet 开发者 wxhere 以及各位 PBH 用户们（排名不分先后）的三方共同的努力下，共同推动了相关 BitComet WebAPI 接口的从无到有，并顺利完成了适配。

在此，我需要向下面的朋友致以诚挚的谢意（排名不分先后）：

* [wxhere](https://www.cometbbs.com/t/210%E6%B5%8B%E8%AF%95%E7%89%88/94132)：维护和开发 BitComet 软件，并添加了 PeerBanHelper 所需要的 WebAPI 接口
* @cxzlw ：从 JS 中完成了关键的 BitComet 新版 WebUI 登陆 AES 加密盐值等部分的计算逆向工程并创建了[Python实现](https://gist.github.com/cxzlw/20a7b5bf649be7b4c3fcff2a796d10bc)，帮助 PeerBanHelper 顺利接入到了 BitComet WebUI；完成了 BitComet 支持配套的前端工作
* @Oniicyan : 在 BitComet 的支持到来前，维护 [BTNScriptBC](https://github.com/Oniicyan/BTNScriptBC) 脚本，帮助 BitComet 用户渡过了一段困难时期，如果你还在使用旧版 BitComet，你也许可以看看这个项目；如果你使用的下载器不受支持，也还可以看看 Ta 的 [BT_BAN](https://github.com/Oniicyan/BT_BAN)。
* [小樱](https://www.itzmx.com/): 参与了 BitComet WebAPI 的测试，帮助我们少踩了许多坑，并提出了许多建设性的建议
* 以及其它所有为 BitComet WebAPI/PeerBanHelper BitComet 支持的推出提供过力量的朋友，谢谢你们！

## 新功能

* 添加了对 [BitComet](https://www.bitcomet.com/en) 下载器的实验性支持 [需要手动设置，请查看文档](https://pbh-btn.github.io/pbh-docs/docs/downloader/BitComet) @cxzlw @Ghost-chu 
  * 需要 v2.10 Beta6 [20240928] 或者更高版本，您可以从[这里下载测试版本](https://www.cometbbs.com/t/210%E6%B5%8B%E8%AF%95%E7%89%88/94132)
* 添加 FreeBSD 包 @Gaojianli 

## 错误修复

* 修复部分页面元素缩放异常的问题 @Gaojianli 

## 已知问题

* 由于接口技术限制，BitComet 的流量数据暂时不会统计在 PeerBanHelper 的流量统计图表中
* BitComet 适配器有时可能出现接口请求超时，但不影响使用。这是 BitComet WebAPI 的一个问题引起的且无法稳定复现。如果你出现了请求超时或者请求消耗大量的时间，欢迎前往 Issue Tracker 提交报告，帮助我们一起检查这个问题

## Docker 用户

DockerHub: `ghostchu/peerbanhelper:v6.4.0`   
阿里云国内镜像加速: `registry.cn-hangzhou.aliyuncs.com/ghostchu/peerbanhelper:v6.4.0`

<hr /><em>This discussion was created from the release <a href='https://github.com/PBH-BTN/PeerBanHelper/releases/tag/v6.4.0'>v6.4.0</a>.</em>

**Comment by cxzlw** [comment]
恭喜~
